common/persistence/visibility/elasticsearch/client/client_v7.go

```diff
@@ -91,6 +91,7 @@ func newSimpleClientV7(url string) (*clientV7, error) {
 	var err error
 	if client, err = elastic.NewClient(
 		elastic.SetURL(url),
+		elastic.SetSniff(false),
```

Comment:
Should this be instead a configurable option?

Reply:
We can make this configurable, but setting this by default shouldn't affect cli commands IMHO.

What do you propose?
Having a flag in all the commands using ES.

Reply:
Yes.  ES flag makes a lot of sense.

Reply:
I think it is ok to have it off. We have it disabled for main ES client and for CLI sniffing doesn't make much sense as every command is short client is short living.